python/observability-online-eval/smolagents/example.py

```
@@ -0,0 +1,104 @@
+for row in rows:
+    stmt = insert(receipts).values(**row)
+    with engine.begin() as connection:
+        connection.execute(stmt)
+
+inspector = inspect(engine)
+columns_info = [(col["name"], col["type"]) for col in inspector.get_columns("receipts")]
+
+table_description = "Columns:\n" + "\n".join([f"  - {name}: {col_type}" for name, col_type in columns_info])
+print(table_description)
+
+from smolagents import tool
```

Ruff (0.12.2)

Module level import not at top of file

(E402)